tests/acceptance/features/webUITextEditor/editTextFiles.feature

```diff
@@ -22,3 +22,20 @@ Feature: textFiles
     And the user closes the text editor
     And the user opens file "New text file.txt" using the webUI
     Then line 1 of the text should be "other text before stuff"
+
+  @issue-core-36233
+  Scenario: Delete and restore hidden text file
```

Comment:
```suggestion
  Scenario: Edit restored hidden text file
```
IMO this is what the scenario is trying to show.

You could also add a separate scenario that just creates a hidden text file, and then edits it with the text editor - that would be useful to have anyway.